### PR TITLE
[BugFix] Forgot to initialize bthread::ExecutionQueueId

### LINKED ISSUE #7763: [Bug] When importing with high concurrency, the import will get stuck
When importing with high concurrency, the import will get stuck

pstack

```
Thread 1531 (Thread 0x7ff1ee007700 (LWP 15468)):
#0  0x00007ff4ec44254d in __lll_lock_wait () from /lib64/libpthread.so.0
#1  0x00007ff4ec43de9b in _L_lock_883 () from /lib64/libpthread.so.0
#2  0x00007ff4ec43dd68 in pthread_mutex_lock () from /lib64/libpthread.so.0
#3  0x0000000001fd3b25 in __gthread_mutex_lock (__mutex=0xedc807b8) at /usr/include/c++/10.3.0/x86_64-pc-linux-gnu/bits/gthr-default.h:749
#4  std::mutex::lock (this=0xedc807b8) at /usr/include/c++/10.3.0/bits/std_mutex.h:100
#5  std::lock_guard<std::mutex>::lock_guard (__m=..., this=<synthetic pointer>) at /usr/include/c++/10.3.0/bits/std_mutex.h:159
#6  starrocks::LoadChannel::cancel (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#7  0x0000000001fd0e12 in starrocks::LoadChannelMgr::cancel (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#8  0x000000000409f82e in brpc::policy::ProcessRpcRequest (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#9  0x0000000004096297 in brpc::ProcessInputMessage (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#10 0x0000000004097143 in brpc::RunLastMessage::operator() (last_msg=0x4972ab00, this=<synthetic pointer>) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/brpc/input_messenger.cpp:141
#11 std::unique_ptr<brpc::InputMessageBase, brpc::RunLastMessage>::~unique_ptr (this=<synthetic pointer>, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/unique_ptr.h:361
#12 brpc::InputMessenger::OnNewMessages (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#13 0x000000000413de0e in brpc::Socket::ProcessEvent (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#14 0x000000000404bd9f in bthread::TaskGroup::task_runner (Python Exception <type 'exceptions.NameError'> Installation error: gdb._execute_unwinders function is missing:
#15 0x00000000041d4581 in bthread_make_fcontext () at /usr/include/c++/10.3.0/typeinfo:100
#16 0x0000000000000000 in ?? ()
```

bthread stack

```
#0  bthread::TaskGroup::sched_to(bthread::TaskGroup**, bthread::TaskMeta*) () at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/task_group.cpp:606
#1  0x000000000404bb8d in bthread::TaskGroup::sched_to (next_tid=<optimized out>, pg=0x7fed3ac329e8) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/task_group_inl.h:78
#2  bthread::TaskGroup::sched(bthread::TaskGroup**) () at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/task_group.cpp:562
#3  0x00000000041d2761 in bthread::butex_wait (arg=<optimized out>, expected_value=expected_value@entry=2, abstime=abstime@entry=0x0) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/butex.cpp:659
#4  0x000000000403df71 in bthread::ExecutionQueueBase::join (id=<optimized out>) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/execution_queue.cpp:225
#5  0x00000000032d65ad in bthread::execution_queue_join<starrocks::vectorized::AsyncDeltaWriter::Task> (id=...) at /root/starrocks/be/src/storage/async_delta_writer.cpp:16
#6  starrocks::vectorized::AsyncDeltaWriter::~AsyncDeltaWriter() () at /root/starrocks/be/src/storage/async_delta_writer.cpp:16
#7  0x0000000001fd6af0 in std::default_delete<starrocks::vectorized::AsyncDeltaWriter>::operator() (__ptr=0xb77f1c3f0, this=0x94aef910) at /usr/include/c++/10.3.0/bits/hashtable_policy.h:2071
#8  std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> >::~unique_ptr (this=0x94aef910, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/unique_ptr.h:361
#9  std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >::~pair (this=0x94aef908, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/stl_pair.h:211
#10 __gnu_cxx::new_allocator<std::__detail::_Hash_node<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, false> >::destroy<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > > > (__p=0x94aef908, this=<optimized out>) at /usr/include/c++/10.3.0/ext/new_allocator.h:156
#11 std::allocator_traits<std::allocator<std::__detail::_Hash_node<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, false> > >::destroy<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > > > (__p=0x94aef908, __a=...) at /usr/include/c++/10.3.0/bits/alloc_traits.h:531
#12 std::__detail::_Hashtable_alloc<std::allocator<std::__detail::_Hash_node<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, false> > >::_M_deallocate_node (__n=0x94aef900,
    this=<optimized out>) at /usr/include/c++/10.3.0/bits/hashtable_policy.h:2053
#13 std::__detail::_Hashtable_alloc<std::allocator<std::__detail::_Hash_node<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, false> > >::_M_deallocate_nodes (
    this=<optimized out>, __n=0x0) at /usr/include/c++/10.3.0/bits/hashtable_policy.h:2075
#14 std::_Hashtable<long, std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, std::allocator<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > > >, std::__detail::_Select1st, std::equal_to<long>, std::hash<long>, std::__detail::_Mod_range_hashing, std::__detail::_Default_ranged_hash, std::__detail::_Prime_rehash_policy, std::__detail::_Hashtable_traits<false, false, true> >::clear (this=0x5544befd8) at /usr/include/c++/10.3.0/bits/hashtable.h:2030
#15 std::_Hashtable<long, std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > >, std::allocator<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > > >, std::__detail::_Select1st, std::equal_to<long>, std::hash<long>, std::__detail::_Mod_range_hashing, std::__detail::_Default_ranged_hash, std::__detail::_Prime_rehash_policy, std::__detail::_Hashtable_traits<false, false, true> >::~_Hashtable (this=0x5544befd8, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/hashtable.h:1387
#16 std::unordered_map<long, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> >, std::hash<long>, std::equal_to<long>, std::allocator<std::pair<long const, std::unique_ptr<starrocks::vectorized::AsyncDeltaWriter, std::default_delete<starrocks::vectorized::AsyncDeltaWriter> > > > >::~unordered_map (this=0x5544befd8, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/unordered_map.h:102
#17 starrocks::LocalTabletsChannel::~LocalTabletsChannel() () at /root/starrocks/be/src/runtime/local_tablets_channel.cpp:40
#18 0x0000000001fd6e62 in starrocks::LocalTabletsChannel::~LocalTabletsChannel (this=0x5544bedc0, __in_chrg=<optimized out>) at /root/starrocks/be/src/runtime/local_tablets_channel.cpp:40
#19 0x0000000001fd4171 in starrocks::RefCountedThreadSafe<starrocks::TabletsChannel, starrocks::DefaultRefCountedThreadSafeTraits<starrocks::TabletsChannel> >::DeleteInternal (x=<optimized out>) at /root/starrocks/be/src/common/status.h:122
#20 starrocks::DefaultRefCountedThreadSafeTraits<starrocks::TabletsChannel>::Destruct (x=<optimized out>) at /root/starrocks/be/src/gutil/ref_counted.h:117
#21 starrocks::RefCountedThreadSafe<starrocks::TabletsChannel, starrocks::DefaultRefCountedThreadSafeTraits<starrocks::TabletsChannel> >::Release (this=<optimized out>) at /root/starrocks/be/src/gutil/ref_counted.h:142
#22 scoped_refptr<starrocks::TabletsChannel>::~scoped_refptr (this=<synthetic pointer>, __in_chrg=<optimized out>) at /root/starrocks/be/src/gutil/ref_counted.h:259
#23 starrocks::LoadChannel::open(brpc::Controller*, starrocks::PTabletWriterOpenRequest const&, starrocks::PTabletWriterOpenResult*, google::protobuf::Closure*) () at /root/starrocks/be/src/runtime/load_channel.cpp:54
#24 0x0000000001fd1dc6 in starrocks::LoadChannelMgr::open(brpc::Controller*, starrocks::PTabletWriterOpenRequest const&, starrocks::PTabletWriterOpenResult*, google::protobuf::Closure*) () at /root/starrocks/be/src/common/closure_guard.h:48
#25 0x000000000409f82e in brpc::policy::ProcessRpcRequest(brpc::InputMessageBase*) () at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/brpc/policy/baidu_rpc_protocol.cpp:496
#26 0x0000000004096297 in brpc::ProcessInputMessage (void_arg=void_arg@entry=0x4972af00) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/brpc/input_messenger.cpp:135
#27 0x0000000004097143 in brpc::RunLastMessage::operator() (last_msg=0x4972af00, this=<synthetic pointer>) at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/brpc/input_messenger.cpp:141
#28 std::unique_ptr<brpc::InputMessageBase, brpc::RunLastMessage>::~unique_ptr (this=<synthetic pointer>, __in_chrg=<optimized out>) at /usr/include/c++/10.3.0/bits/unique_ptr.h:361
#29 brpc::InputMessenger::OnNewMessages(brpc::Socket*) () at /usr/include/c++/10.3.0/bits/unique_ptr.h:355
#30 0x000000000413de0e in brpc::Socket::ProcessEvent(void*) () at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/brpc/socket.cpp:1017
#31 0x000000000404bd9f in bthread::TaskGroup::task_runner(long) () at /var/local/thirdparty/src/incubator-brpc-0.9.7/src/bthread/task_group.cpp:296
#32 0x00000000041d4581 in bthread_make_fcontext () at /var/local/gcc/x86_64-pc-linux-gnu/libstdc++-v3/include/bits/shared_ptr_base.h:134
#33 0x0000000000000000 in ?? ()
```

### Steps to reproduce the behavior (Required)
1. CREATE TABLE '...'
2. INSERT INTO '....'
3. SELECT  '....'

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

Fix #7763

### CODE REVIEW

**Review by imay — DISMISSED** [review]

File: be/src/storage/async_delta_writer.h [hunk 1]
```diff
@@ -27,7 +27,8 @@ class AsyncDeltaWriter {
     // Create a new transaction in TxnManager and return a AsyncDeltaWriter for write.
     static StatusOr<std::unique_ptr<AsyncDeltaWriter>> open(const DeltaWriterOptions& opt, MemTracker* mem_tracker);
 
-    AsyncDeltaWriter(private_type, std::unique_ptr<DeltaWriter> writer) : _writer(std::move(writer)) {}
```
Comment (trueeyu) [inline be/src/storage/async_delta_writer.h:30]: 0 is a meaningful queue id

Comment (trueeyu) [reply be/src/storage/async_delta_writer.h:30]:
```
       m->_this_id = make_id(
                _version_of_vref(m->_versioned_ref.fetch_add(
                                    1, butil::memory_order_release)), slot); 
```

**Review by trueeyu — DISMISSED** [review]

**Review by trueeyu — DISMISSED** [review]

**Review by trueeyu — DISMISSED** [review]

File: be/src/storage/lake/async_delta_writer.cpp [hunk 2]
```diff
@@ -26,10 +26,13 @@ class AsyncDeltaWriterImpl {
 public:
     using Callback = AsyncDeltaWriter::Callback;
 
+    // Undocemented rule of bthread that -1(0xFFFFFFFFFFFFFFFF) is an invalid ExecutionQueueId
+    constexpr static uint64_t kInvalidQueueId = (uint64_t)-1;
```
Comment (chaoyli) [inline be/src/storage/lake/async_delta_writer.cpp:30]: already defined in async_delta_writer.cpp.h

**Review by chaoyli — DISMISSED** [review]

**Review by trueeyu — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Comment by trueeyu** [comment]
run starrocks_clang-format

**Pull request merged**